packages/components/src/hooks/UIProvider.tsx

```diff
@@ -8,6 +8,11 @@ export interface Toast {
   icon?: ReactNode
 }
 
+export interface Popover {
```

Comment:
To avoid unexpected changes in the payload, I've decided to create this new type to handle the popover.